Resources/Prototypes/Entities/Mobs/Species/reptilian.yml

```
@@ -93,17 +103,35 @@
+          32:
+            sprite: _White/Mobs/Species/displacement.rsi
+            state: jumpsuit
+      shoes:
+        sizeMaps:
+          32:
+            sprite: _White/Mobs/Species/displacement.rsi
+            state: shoes
+    # WWDP-Edit-End
 
-#Weh
+#Weh
```

YAMLlint (1.35.1)

[warning] missing starting space in comment

(comments)